### PR TITLE
Feature/filter speeddir2uv rh2q

### PR DESCRIPTION
**Pull request by mpvginde**
adds new filters:
- wind-speed and -direction <--> u- and v-wind on single levels
- relative humidity <--> specific humidity on single levels
- relative humidity <--> dewpoint temperature on single levels
- relative humidity <--> specific humidity on pressure levels

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/anemoi-datasets?pullRequest=40) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/anemoi-datasets?pullRequest=40) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/anemoi-datasets?pullRequest=40) it.</sub>

File: src/anemoi/datasets/create/functions/filters/single_level_relative_humidity_to_specific_humidity.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    result = FieldArray()
+
+    MANDATORY_KEYS = ["A", "B"]
+    OPTIONAL_KEYS = ["t_ml", "q_ml"]
+    MISSING_KEYS = []
+    DEFAULTS = dict(t_ml="t", q_ml="q")
+
+    for key in OPTIONAL_KEYS:
+        if key not in kwargs:
+            print(f"key {key} not found in yaml-file, using default key: {DEFAULTS[key]}")
```
Comment (mpvginde) [inline src/anemoi/datasets/create/functions/filters/single_level_relative_humidity_to_specific_humidity.py:26]: This `print` should probably be replaced by a log-message

File: src/anemoi/datasets/create/functions/filters/single_level_specific_humidity_to_relative_humidity.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+    result = FieldArray()
+
+    MANDATORY_KEYS = ["A", "B"]
+    OPTIONAL_KEYS = ["t_ml", "q_ml"]
+    MISSING_KEYS = []
+    DEFAULTS = dict(t_ml="t", q_ml="q")
+
+    for key in OPTIONAL_KEYS:
+        if key not in kwargs:
+            print(f"key {key} not found in yaml-file, using default key: {DEFAULTS[key]}")
```
Comment (mpvginde) [inline src/anemoi/datasets/create/functions/filters/single_level_specific_humidity_to_relative_humidity.py:233]: replace `print` by log-message

**Review by b8raoult — APPROVED** [review]

File: src/anemoi/datasets/create/functions/filters/single_level_specific_humidity_to_relative_humidity.py (hunk 2 above)
Comment (b8raoult) [reply src/anemoi/datasets/create/functions/filters/single_level_specific_humidity_to_relative_humidity.py:239]: I'll do it.

**Review by b8raoult — APPROVED** [review]

File: src/anemoi/datasets/create/functions/filters/single_level_relative_humidity_to_specific_humidity.py (hunk 1 above)
Comment (b8raoult) [reply src/anemoi/datasets/create/functions/filters/single_level_relative_humidity_to_specific_humidity.py:32]: I'll do it.

**Pull request merged**